_posts/2022-04-28-cockpit-268.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cockpit 268
+author: garrett
+date: '2022-04-28'
+tags: cockpit
+slug: cockpit-268
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 268:
+
+
+## Software Updates: Only install kpatches
+
+Live patch a kernel without having to reboot or restart services with the new "Install kpatch updates" button. This allows patching of urgent kernel vulnerabilities without rebooting.
+
+![kpatch]({{ site.baseurl }}/images/268-software-updates-only-install-kpatches.png)
+
+
+## Try it out
+
+Cockpit 268 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/268)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
```

Comment:
We still need the URLs. Bodhi's latest (at the above URLs) is still 267.

Reply:
Yup, but they are somehow missing from bodhi.

Reply:
Turns out  we tried to release the wrong version to bodhi:

```
> Completed job
"$@"
/usr/lib64/python3.10/getpass.py:91: GetPassWarning: Can not control echo on the terminal.
  passwd = fallback_getpass(prompt, stream)
Warning: Password input may be echoed.
Password: 
Build does not exist: cockpit-268.1-2.fc35
You may not specify an empty list of builds.
ACL validation mechanism was unable to determine ACLs.
> Completed job
"$@"
Build does not exist: cockpit-268.1-2.fc36
You may not specify an empty list of builds.
ACL validation mechanism was unable to determine ACLs.
> Completed job
"$@"
> Pushing changes to guide
To github.com:cockpit-project/cockpit-project.github.io
   b1591be..3650615  main -> main
> Completed job
```